apps/web/app/api/recorded-daily-video/route.ts

```diff
@@ -5,8 +5,10 @@ import type { NextRequest } from "next/server";
 import { NextResponse } from "next/server";
 
 import { getRoomNameFromRecordingId, getBatchProcessorJobAccessLink } from "@calcom/app-store/dailyvideo/lib";
-import { sendDailyVideoRecordingEmails } from "@calcom/emails";
-import { sendDailyVideoTranscriptEmails } from "@calcom/emails";
```

Comment:
importing from this barrel file exposed this API route to client-code (`Trans` from `next-i18next`)